src/Standards/Squiz/Docs/WhiteSpace/SuperfluousWhitespaceStandard.xml

```diff
@@ -0,0 +1,66 @@
+<documentation title="Superfluous Whitespace">
+    <standard>
+    <![CDATA[
+    There should be no extra whitespace at the start of a file.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No whitespace preceding first content in file.">
+        <![CDATA[<?php
+echo 'hi';
+]]>
+        </code>
+        <code title="Invalid: Whitespace used before content in file.">
+        <![CDATA[
+        <em></em>
+<?php
+echo 'hi';
+]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There should be no trailing whitespace at the end of lines.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No whitespace found at end of line.">
+        <![CDATA[
+echo 'hello';
+]]>
+        </code>
+        <code title="Invalid: Whitespace found at end of line.">
+        <![CDATA[
+echo 'hello';<em>   </em>
+]]>
+        </code>
```

Comment:
When looking at these code sample in the command line, they don't look any different. That can't really be helped, but I wonder if it would clarify things if instead of "hello", the text used would describe what's happening ?

Something like "semicolon followed by new line char" vs "trailing spaces after semicolon" ?

Reply:
That makes a lot of sense. I've reworded all of the code examples in aa088bdfc